envConfig/live.env

```diff
@@ -5,6 +5,7 @@ SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/include/articles/public/
 SIMORGH_ATI_BASE_URL=https://a1.api.bbc.co.uk/hit.xiti?
 SIMORGH_CONFIG_URL=https://config.api.bbci.co.uk/
 SIMORGH_INCLUDES_BASE_URL=https://www.bbc.com/ws/includes
+SIMORGH_INCLUDES_BASE_AMP_URL=https://news.files.bbci.co.uk
```

Comment:
this is still the same as the `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` env, any way we could reassign that (to be explicit)? or reuse if possible ?

Reply:
@andrewscfc commented:
> So we need this because `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` points to localhost [in local.env](https://github.com/bbc/simorgh/blob/latest/envConfig/local.env#L3). That makes sense because Simorgh hosts our static assets on a local build.
> 
> In this case we a referencing static assets that Simorgh does not manage, I'd personally go back to adding a specific `.env` var for includes as on this PR: [#6873 (comment)](https://github.com/bbc/simorgh/pull/6873#discussion_r442248006) ; there is a distinction between static assets Simorgh manages and those managed elsewhere.

Reply:
thanks @Simon2828